### PR TITLE
docs: resolve `preloadRouteComponents` page heading error

### PR DESCRIPTION
**Pull request by Mini-ghost**
### 🔗 Linked issue

N/A

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

<https://nuxt.com/docs/api/utils/preload-route-components>
The section heading of this page should be `preloadRouteComponents`.

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/3.api/3.utils/preload-route-components.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: "preloadRouteComponents"
 ---
 
-# `preloadComponents`
+# `preloadRouteComponents`
 
 `preloadRouteComponent` allows you to manually preload individual pages in your Nuxt app.
```
Comment (manniL) [inline docs/3.api/3.utils/preload-route-components.md:7]: While we are at it, 
```suggestion
`preloadRouteComponents` allows you to manually preload individual pages in your Nuxt app.
```

**Review by TheAlexLichter — APPROVED** [review]
Good catch 👍🏻

**Comment by Mini-ghost** [comment]
@manniL I didn't find that there is also a typo in the content, do you want to modify it together?

**Comment by manniL** [comment]
> @manniL I didn't find that there is also a typo in the content, do you want to modify it together?

Just committed my suggestion (missing `s`) so all good 👍🏻

**Review by danielroe — APPROVED** [review]

**Pull request merged**